src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceFileLogger.cs

```diff
@@ -87,7 +90,11 @@ public virtual async Task ProcessLogQueue(object state)
             while (!_cancellationTokenSource.IsCancellationRequested)
             {
                 await InternalProcessLogQueue();
-                await Task.Delay(TimeSpan.FromSeconds(FlushFrequencySeconds), _cancellationTokenSource.Token).ContinueWith(task => { });
+                await Task.Delay(TimeSpan.FromSeconds(CurrentFlushFrequencySeconds), _cancellationTokenSource.Token).ContinueWith(task => { });
+                if ( CurrentFlushFrequencySeconds < MaxFlushFrequencySeconds)
+                {
+                    CurrentFlushFrequencySeconds++;
```

Comment:
Did we mean to increase this one by one? Is that too aggressive? Should use a growth factor value to multiply this value instead? (ex: multiply by 2 every time, so it will be 1, 2, 4, 8, 16 ).  Also we can make this growth factor value overridable via config so that we can play with different values to see which one is the correct fit to set as default.  Very much possible you already evaluated that option. But wanted to share this thought.